core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -139,19 +131,16 @@ type EthConfirmer struct {
 }
 
 // NewEthConfirmer instantiates a new eth confirmer
-func NewEthConfirmer(db *sqlx.DB, ethClient evmclient.Client, config Config, keystore KeyStore,
+func NewEthConfirmer(orm ORM, ethClient evmclient.Client, config Config, keystore KeyStore,
```

Comment:
pass in ORM struct into NewEthConfirmer and remove presence of sqlx.DB and pg.Q in the eth confirmer